pkg/api/v1alpha1/edgeconnect/edgeconnect_types.go

```diff
@@ -187,3 +189,27 @@ type EdgeConnectList struct { //nolint:revive
 func init() {
 	v1alpha1.SchemeBuilder.Register(&EdgeConnect{}, &EdgeConnectList{})
 }
+
+func (e *EdgeConnect) HostPatterns(k8sHostname string) []string {
```

Comment:
Does it make sense to pass in the `k8sHostName` if it is always the value of `e.K8sAutomationHostPattern()`?

You can just call it inside the method?

Reply:
I'm stupid, yes, totally true
[Fixed here](https://github.com/Dynatrace/dynatrace-operator/pull/3332/commits/04e9a7c0cd1842edf9e28cac77a9f3d192c530a2)! Thanks